hw02/hw02_exploring_gapminder.Rmd

```diff
@@ -0,0 +1,65 @@
+library(gapminder)
+library(ggplot2)
+library(palmerpenguins)
+library(tidyverse)
+library(dplyr)
+
```

Comment:
Great job on your work! One thing you should include for next time is title, author, date, and output at the top.